jekyll/_cci2/server-3-install.adoc

```diff
@@ -3,681 +3,183 @@ version:
-it is often used with implicit TLS instead of StartTLS. Server only supports StartTLS for encrypted submission. +
-
-WARNING: Outbound connections on port 25 are blocked on most cloud providers. Should you select this port, be aware that
-your notifications may fail to send.
-
-[start=5]
-. *Enable StartTLS*: Enabling this will encrypt mail submission. +
+```bash
+kubectl kots install circleci-server --http-proxy <http-proxy-uri> --https-proxy <https-proxy> --no-proxy <no proxy list>
+```
 
-WARNING: You should only disable this if you can otherwise guarantee the confidentiality of traffic.
+The load balancer endpoints must be added to the no-proxy list for the following services: `output processor` and `vm-service`. This is because the no-proxy list is shared between the application and build-agent. The application and build-agent are assumed to be behind the same firewall and therefore cannot have a proxy between them. 
 
-### VM Service
-VM Service configures VM and remote docker jobs. You can configure a number of options for VM service, such as scaling rules.
+For further information see the https://circleci.com//docs/2.0/server-3-operator-proxy/index.html[Configuring a Proxy] guide.
 
-NOTE: We recommend that you leave these options at their defaults until you have successfully configured and verified your server installation.
+=== Frontend Settings 
+Frontend settings control the web application specific aspects of the CircleCI system. 
 
-#### Authentication and Permissions
-##### AWS EC2
-You will need the following fields to configure your VM Service to work with AWS EC2. Note that the Access Key and Secret
-Key used by VM Service differs from the policy used by Object Storage in the previous section. VM Service and Object Storage are kept distinct to allow organizations to utilize different cloud and on-premise providers within the same installation. 
+Complete the following fields: 
 
-. *AWS Region* (required): This is the region the application is in.
-. *AWS Windows AMI ID* (optional): If you require Windows builders, you can supply an AMI ID for them here.
-. *Subnet ID* (required): Choose a subnet (public or private) where the VMs should be deployed.
-. *Security Group ID* (required): This is the security group that will be attached to the VMs. It must be created manually.
+* *Domain Name (required)* - Enter the domain name you specified when creating your Frontend TLS key and certificate. 
 
-The recommended security group configuration can be found in the xref:server-3-install-hardening-your-cluster.adoc#external-vms[Hardening Your Cluster] section. Additionally, the below commands can be run to create the necessary security groups in AWS or GCP.
+* *Frontend Replicas (optional)* - Used to increase the amount of traffic that can be handled by the frontend. 
 
-AWS
+* *Frontend TLS Private Key (required)* - You created this during your pre-requisite steps. You can retrieve this value with the following command: `
```

Comment:
@nathanfishcircleci @ryanwohara These are the settings where we need to clear up what we are recommending people do. If they check the recommended setting "Automatically request and manage ..."  then they don't need to add the private key and certificate here... correct?

Reply:
That is correct. While having a cert is currently required, they can have LetsEncrypt manage it automatically via Traefik. I personally generate the cert using LetsEncrypt's certbot tool and upload the cert manually, so customers have that option.

Reply:
OK I'll leave it how it is for now, thanks!